src/main/java/com/radixdlt/crypto/Hash.java

```diff
@@ -100,10 +101,9 @@ public static Hash of(byte[] dataToBeHashed)	{
 	}
 
 
-	private final byte[] 	data;
-	private Supplier<EUID>	idCached = Suppliers.memoize(this::computeId);
-
-	private final int hashCodeCached;
+	private final byte[] data;
+	private final transient Supplier<EUID> idCached = Suppliers.memoize(this::computeId);

```

Comment:
These don't need to be transient.

Reply:
I do know `transient` is used by other serialization libs (e.g. gson) so thought it was good indicator whether parameter should in general be cached. But no big deal, can easily remove.

Reply:
OK, no problems.  Let's leave it as it is then.